### PR TITLE
Repackage -  Apache Log4j Vulnerability Detection , Web Session Essen…

### PR DESCRIPTION
**Pull request by v-rusraut**
Change(s):
   - Repackaging

   Reason for Change(s):
   - Tagged for dependent solutions for deployment

   Version Updated:
   - Apache Log4j Vulnerability Detection (3.0.1)
   - Web Session Essentials (3.0.1)
   - Network Session Essentials (3.0.1)

   Testing Completed:
   - Done
   **Apache Log4j Vulnerability Detection**
ApacheLog4

**Web Session Essentials**
WebSessionEssentials

**Network Session Essentials**
NetworkSessionEssentials

### CODE REVIEW

File: Solutions/Network Session Essentials/Hunting Queries/DetectPortMisuseByAnomalyHunting.yaml [hunk 1]
```diff
@@ -1,6 +1,9 @@
 id: 906c20c6-b62c-4af7-be91-d7300e3bded2
 name: Detect port misuse by anomaly (ASIM Network Session schema)
 description: |
+  'Detects anomalous port usage with ASIM normalization and is applied to sources supporting ASIM Network Session schema. Tune with 'NetworkSession_Monitor_Configuration' watchlist. The query uses summarized data for performance.'
```
Comment (rahul0216) [inline Solutions/Network Session Essentials/Hunting Queries/DetectPortMisuseByAnomalyHunting.yaml:4]: "This hunting query detect anomalous pattern in port usage with ASIM normalization. To tune the query to your environment configure it using the 'NetworkSession_Monitor_Configuration' watchlist."

File: Solutions/Apache Log4j Vulnerability Detection/Hunting Queries/WAF_log4j_vulnerability.yaml [hunk 2]
```diff
@@ -1,6 +1,8 @@
 id: 1d4d383e-0ca6-4d3a-a861-8f37aeef18cb
 name: Azure WAF Log4j CVE-2021-44228 hunting
 description: |
+   'This hunting query looks in Azure Web Application Firewall data to find possible exploitation attempts for CVE-2021-44228 involving log4j vulnerability.'
```
Comment (rahul0216) [inline Solutions/Apache Log4j Vulnerability Detection/Hunting Queries/WAF_log4j_vulnerability.yaml:4]: "This hunting query searches possible exploitation attempts for CVE-2021-44228 involving log4j vulnerability in Azure Web Application Firewall logs."

File: Solutions/Apache Log4j Vulnerability Detection/Hunting Queries/NetworkConnectionldap_log4j.yaml [hunk 3]
```diff
@@ -1,6 +1,8 @@
 id: 19abc034-139e-4e64-a05d-cb07ce8b003b
 name: Malicious Connection to LDAP port for CVE-2021-44228 vulnerability
 description: |
+   'This query detects exploitation attempts for CVE-2021-44228 involving log4j vulnerability by looking for connections to default LDAP ports. Log4j is an open-source Apache logging library used in many Java-based applications'
```
Comment (rahul0216) [inline Solutions/Apache Log4j Vulnerability Detection/Hunting Queries/NetworkConnectionldap_log4j.yaml:4]: This is enough ' This query detects exploitation attempts for CVE-2021-44228 involving log4j vulnerability by looking for connections to default LDAP ports. '

File: Solutions/Apache Log4j Vulnerability Detection/Hunting Queries/Linux_Toolkit_Detected.yaml [hunk 4]
```diff
@@ -1,8 +1,9 @@
 id: 09e45ec6-ac42-4b5a-be69-54623c4aa062
 name: Possible Linux attack toolkit detected via Syslog data
 description: |
-  'This query uses syslog data to alert on any attack toolkits associated with massive scanning or exploitation attempts against a known vulnerability.
-  Attackers may perform such operations as seen recently to exploit the remote code execution vulnerability in Log4j component of Apache to scope and prioritize post-compromise objectives.
+   'This query alerts on attack toolkits associated with massive scanning or exploitation attempts against a known vulnerability. Attackers may exploit remote code execution vulnerability in Log4j component of Apache to prioritize post-compromise objectives'
```
Comment (rahul0216) [inline Solutions/Apache Log4j Vulnerability Detection/Hunting Queries/Linux_Toolkit_Detected.yaml:4]: 'This query searches for usage of attack toolkits associated with massive scanning or exploitation of remote code execution vulnerability in Log4j component of Apache.'

**Review by rahul0216 — COMMENTED** [review]
I have few suggestions for the description inline. Please have a look.

File: Solutions/Apache Log4j Vulnerability Detection/Hunting Queries/Container_Miner_Activity.yaml [hunk 5]
```diff
@@ -1,6 +1,8 @@
 id: 6fee32b3-3271-4a3f-9b01-dbd9432a1707
 name: Possible Container Miner related artifacts detected
 description: |
+  'Query uses syslog data to alert on artifacts from container images used in digital cryptocurrency mining, often seen post compromise post CVE-2021-44228 Log4j vulnerability exploitation and prioritize post-compromise objectives.'
```
[the remark this answers is not in the document]
Comment (v-rusraut) [reply Solutions/Apache Log4j Vulnerability Detection/Hunting Queries/Container_Miner_Activity.yaml]: Resolved

File: Solutions/Apache Log4j Vulnerability Detection/Hunting Queries/Firewall_Disable_Activity.yaml [hunk 6]
```diff
@@ -1,6 +1,8 @@
 id: e178baf5-3cf3-4960-8ca4-8da6d90d8206
 name: Suspicious manipulation of firewall detected via Syslog data
 description: |
+  'This query uses syslog data, Attackers often manipulate firewall to evade defenses using syslog data. They exploit the remote code execution vulnerability in Log4j component of Apache for C2 communications or exfiltration. '
```
[the remark this answers is not in the document]
Comment (v-rusraut) [reply Solutions/Apache Log4j Vulnerability Detection/Hunting Queries/Firewall_Disable_Activity.yaml]: Resolved

File: Solutions/Network Session Essentials/Hunting Queries/DetectPortMisuseByAnomalyHunting.yaml (hunk 1 above)
Comment (v-rusraut) [reply Solutions/Network Session Essentials/Hunting Queries/DetectPortMisuseByAnomalyHunting.yaml]: Resolved

File: Solutions/Apache Log4j Vulnerability Detection/Hunting Queries/WAF_log4j_vulnerability.yaml (hunk 2 above)
Comment (v-rusraut) [reply Solutions/Apache Log4j Vulnerability Detection/Hunting Queries/WAF_log4j_vulnerability.yaml]: Resolved

File: Solutions/Apache Log4j Vulnerability Detection/Hunting Queries/NetworkConnectionldap_log4j.yaml (hunk 3 above)
Comment (v-rusraut) [reply Solutions/Apache Log4j Vulnerability Detection/Hunting Queries/NetworkConnectionldap_log4j.yaml]: Resolved

File: Solutions/Apache Log4j Vulnerability Detection/Hunting Queries/Linux_Toolkit_Detected.yaml (hunk 4 above)
Comment (v-rusraut) [reply Solutions/Apache Log4j Vulnerability Detection/Hunting Queries/Linux_Toolkit_Detected.yaml]: Resolved

**Review by rahul0216 — DISMISSED** [review]
Description changes looks good.

**Review by v-atulyadav — APPROVED** [review]

**Pull request merged**